.github/workflows/build-push-images.yaml

```diff
@@ -23,26 +23,41 @@ jobs:
         run: |
           PACKAGE_DIR="./deploy/olm-catalog/community-kubevirt-hyperconverged"
           CSV_VERSION=$(ls -d ${PACKAGE_DIR}/*/ | sort -rV | awk "NR==$((RELEASE_DELTA+1))" | cut -d '/' -f 5)
-          echo "CSV_VERSION=${CSV_VERSION}-unstable" >> $GITHUB_ENV
+          echo "CSV_VERSION=${CSV_VERSION" >> $GITHUB_ENV
```

Comment:
you didn't close the curly parenthesis ('`}`'), which will cause the workflow to crash